examples/full_featured/plotly-example-full.agent.nut

```diff
@@ -0,0 +1,275 @@
+// Copyright (c) 2015 Electric Imp
+// This file is licensed under the MIT License
+// http://opensource.org/licenses/MIT
+
+class Plotly {
+    
```

Comment:
We're going to pull this in as v1.0.0, so please add `static version [1, 0, 0];`